<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WKeyWordGroup</class>
 <widget class="QWidget" name="WKeyWordGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关键词设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>群消息关键词(触发后将发送一封邮件)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="check_streng">
       <property name="text">
        <string>强力检测</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="list_keyWord"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edit_keyWord"/>
     </item>
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
